### PR TITLE
Annotation: read correct volume-data-zip from nml

### PR DESCRIPTION
**Pull request by jstriebel**
### Description:
- Uses the volume.location info from the annotation's nml to find the correct volume data zipfile.

### Todos:
 - [ ] Updated Changelog

### CODE REVIEW

File: webknossos/webknossos/annotation/annotation.py [hunk 1]
```diff
@@ -61,8 +71,10 @@ def save_volume_annotation(
     ) -> Layer:
         # todo pylint: disable=fixme
         # the name is about to change with multiple volume annotations
```
Comment (fm3) [inline webknossos/webknossos/annotation/annotation.py:73]: the comment can go now :)

**Review by fm3 — APPROVED** [review]
Thanks for jumping in so quickly!
LGTM, but I did not test this separately. Did you test it or should I set it up on my side?

File: webknossos/webknossos/annotation/annotation.py (hunk 1 above)
Comment (jstriebel) [inline webknossos/webknossos/annotation/annotation.py:73]:
```suggestion
```

**Comment by jstriebel** [comment]
>Did you test it or should I set it up on my side? 

@fm3 I tested it with an annotation that still uses `data.zip` and a new one with `data_Volume.zip` and both work fine :+1:

File: webknossos/webknossos/annotation/annotation.py [hunk 2]
```diff
@@ -60,9 +70,10 @@ def save_volume_annotation(
         self, dataset: Dataset, layer_name: str = "volume_annotation"
     ) -> Layer:
         # todo pylint: disable=fixme
```
Comment (philippotto) [inline webknossos/webknossos/annotation/annotation.py:72]:
```suggestion
        # todo pylint: disable=fixme
```

**Pull request merged**